Apps/WebAPI/sdLitica.WebAPI/Services/InfluxDBService.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols;
+using sdLitica.Helpers;
+using sdLitica.WebAPI.Services;
+using Vibrant.InfluxDB.Client;
+using Vibrant.InfluxDB.Client.Rows;
+
+namespace sdLitica.WebAPI.Services
+{
+    public class InfluxDbService : IInfluxDB
```

Comment:
Please, do not reference InfluxDB packages in this project. Encapsulate it in a new project in the Persistance Layer. The new project can be named as: ``sdLitica.TimeSeries``. Also, the implementation of the InfluxService should be in the new project.